src/main/java/app/pedidos/WebApp/domain/usecase/ProcessPaymentCreditCardUseCase.java

```diff
@@ -0,0 +1,16 @@
+package app.pedidos.WebApp.domain.usecase;
+
+import app.pedidos.WebApp.domain.gateway.ProcessPaymentCreditGateway;
+import app.pedidos.WebApp.domain.models.CreditCard;
+import app.pedidos.WebApp.domain.models.PaymentStatus;
+
+public class ProcessPaymentCreditCardUseCase {
```

Comment:
Essa classe tb não faz nada, não devemos permitir, pois se for para produção assim, pode gerar erros. Cria outra task no github e abra um novo PR.